### PR TITLE
Creating Components

### PR DESCRIPTION
**Pull request by ghost**
- Added Image folder 
- Created Home Component
- Created Navbar Component

### CODE REVIEW

File: Front-End/speaky/README.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
+# Getting Started with Create React App
```
Comment (PatelNeelMahesh) [inline Front-End/speaky/README.md]: Remove the default readme content and add details, figma file about your capstone.

File: Front-End/speaky/src/setupTests.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+// jest-dom adds custom jest matchers for asserting on DOM nodes.
```
Comment (PatelNeelMahesh) [inline Front-End/speaky/src/setupTests.js]: You can remove this file.

File: Front-End/speaky/src/reportWebVitals.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
+const reportWebVitals = onPerfEntry => {
```
Comment (PatelNeelMahesh) [inline Front-End/speaky/src/reportWebVitals.js]: You can remove this file.

**Review by PatelNeelMahesh — COMMENTED** [review]
Make the required changes before merging the pull request.

**Review by PatelNeelMahesh — APPROVED** [review]
You can merge pull request.

**Comment by Itachi-789** [comment]
Creating Components

**Pull request merged**